cmd/admin/block.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/coinbase/chainstorage/internal/blockchain/jsonrpc"
 	"github.com/coinbase/chainstorage/internal/blockchain/parser"
 	"github.com/coinbase/chainstorage/internal/config"
-	"github.com/coinbase/chainstorage/internal/storage/blobstorage/downloader"
+	downloader "github.com/coinbase/chainstorage/internal/storage/block_downloader"
```

Comment:
Yes, you're right. The downloader should be part of BlobStorage because it's a BlobStorage downloader. So I'm reverting the second commit.

However, BlobStorage is used in the server and the downloader is used in the client, these 2 modules should be independent.